lib/tasks/rbs.rake

```diff
@@ -2,7 +2,7 @@ if Rails.env.local?
   require "rbs_rails/rake_task"
 
   namespace :rbs do
-    task setup: %i[clean collection generate load_routes rbs_rails:all subtract]
+    task setup: %i[clean collection generate load_routes rbs_rails:all validate]
```

Comment:
It's better to run `rbs:validate` at the end of `rbs:setup` to validate all types.